packages/compass-import-export/src/modules/import.ts

```diff
@@ -280,17 +261,37 @@ export const startImport = () => {
     const abortSignal = abortController.signal;
     dispatch(onStarted(abortController));
 
-    let promise;
+    openToast(importToastId, {
+      dataTestId: 'import-toast-in-progress',
+      title: `Importing ${fileType} file...`,
```

Comment:
Should we pass a filename here or something to keep some context about what's getting imported?

Reply:
Nice suggestion, I was using the type from the designs, but I like the sound of the filename instead of just the file type. I'll double check with product before updating.

Reply:
Updated 👍 
![Screenshot 2023-03-27 at 11 00 20 AM](https://user-images.githubusercontent.com/1791149/227980305-3b926dc4-24df-411d-813e-10e01ee02180.png)